gist.py

```diff
@@ -216,8 +214,8 @@ def on_gist_filename(filename):
                 sublime.status_message("%s Gist: %s" % (self.mode(), gist_html_url))
 
                 if gistify:
-                    gistify_view(self.view, gist, list(gist['files'].keys())[0])
-                # else:
+                    helpers.gistify_view(self.view, gist, list(gist['files'].keys())[0])
+                    # else:
                     # open_gist(gist['url'])
 
```

Comment:
Really @sanchopanca ? You changed this indentation manually? Looks like editor autocorrection. Sorry.